ext/afform/core/afform.php

```diff
@@ -179,7 +179,7 @@ function _afform_hook_civicrm_angularModules($e) {
       // Permissions needed for conditionally displaying edit-links
       'permissions' => [
         'manage own afform',
-        'administer search_kit',
+        'manage own search_kit',
```

Comment:
Keep in mind that the `permissions` array in an angular module declaration is just a lookup, not an enforcement. So this line is setting a JS variable. In this case, changing this line requires also changing the js code which uses the variable:
https://github.com/civicrm/civicrm-core/blob/df418e91e6ba2b2511d270aeb977505804c2f360/ext/afform/admin/Civi/AfformAdmin/AfformAdminInjector.php#L72

Reply:
Ok, so ideally we should just append to the list , or is it ok to also change the permission `false` option to match the new permission type?

Reply:
Change made